projects/project-1/assignment/assignment_1_short.md

```diff
@@ -0,0 +1,3235 @@
+
+|        | Funct. | iFunct. | Trans. | Symm. | Asymm. | Ref. | Irref. |
+|--------|--------|---------|--------|-------|--------|------|--------|
+| **Funct.**   | -      | OK      | X<sup>NS</sup>    | OK    | OK     | OK   | OK     |
+| **iFunct.**  | OK     | -       | X<sup>NS</sup>    | OK    | OK     | OK   | OK     |
+| **Trans.**   | X<sup>NS</sup>    | X<sup>NS</sup>     | -      | OK    | X<sup>NS</sup>    | OK   | X<sup>NS</sup>    |
+| **Symm.**    | OK     | OK      | OK     | -     | X<sup>UNSAT</sup> | OK   | OK     |
+| **Asymm.**   | OK     | OK      | X<sup>NS</sup>    | X<sup>UNSAT</sup>| -      | X<sup>UNSAT</sup>| OK    |
+| **Ref.**     | OK     | OK      | OK     | OK    | X<sup>UNSAT</sup> | -    | X<sup>UNSAT</sup> |
+| **Irref.**   | OK     | OK      | X<sup>NS</sup>    | OK    | OK     | X<sup>UNSAT</sup>| -      |
+
+\
+Trans.-Funct (NS)
```

Comment:
It’s a bit unclear what NS stands for. It would be good to explicitly define this term for the reader (assuming NS stands for "Not Satisfiable" or "Not Decidable")

Reply:
I think the explanation for Irref-Transitivity is missing